### ISSUE TITLE
[FEATURE] Support Rails7 model varients

### ISSUE THREAD

**Issue by tarellel** [rootstrap/active-storage-base64#68]
Using the rails7 model variants errors out the app.

```ruby
# model
has_one_base64_attached :avatar do |attachable|
   attachable.variant :thumb, resize: "100x100"
end

# view
image_tag(@user&.avatar&.variant(:thumb), class: 'inline-block h-7 w-7 rounded-full', loading: 'lazy')
```

Only the current rails 6 method works to create a variant when pulling the attachment in a view.

```ruby
image_tag(@user&.avatar&.variant(format: :jpg, strip: true, quality: 80, resize_to_fill: [@size, @size]), class: 'inline-block h-7 w-7 rounded-full', loading: 'lazy')
```

**Comment by AhmedKamal20** [comment]
does even v1.2.0 support rails 7 now?

**Comment by santib** [comment]
> does even v1.2.0 support rails 7 now?

@AhmedKamal20 no, I'm about to release v2.0.0 with support to rails 7 soon 😄 see https://github.com/rootstrap/active-storage-base64/pull/69

**Comment by santib** [comment]
@tarellel @AhmedKamal20 I just published v2.0.0 of the gem with support for Rails 7. Enjoy 😄

**Comment by tarellel** [comment]
@santib 
Thank you very much, that was quite an amazing turn around.
